CHANGELOG.md

```diff
@@ -319,4 +334,6 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 - Initial release of the puppet extension.
 
+[0.18.0]: https://github.com/lingua-pupuli/puppet-vscode/compare/0.18.0..0.17.0
```

Comment:
Need an entry for `Unreleased`

Reply:
The versions here and the line below should be flipped.

Reply:
Fixed